junit-jupiter-api/src/main/java/org/junit/jupiter/api/DisplayNameGenerator.java

```diff
@@ -141,4 +143,97 @@ private static boolean hasParameters(Method method) {
 		}
 	}
 
+	/**
+	 * {@code DisplayNameGenerator} that generate complete sentences.
```

Comment:
```suggestion
	 * {@code DisplayNameGenerator} that generates complete sentences.
```

Reply:
Thanks for the typo correction, I'll be more careful on that